### PR TITLE
New background for landing page

### PR DESCRIPTION
**Pull request by JohanBjerning**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *qlikcore* ready!

Built with commit c3ac39627f3acecfdf93aa7149f5aead820ce2d7

https://deploy-preview-399--qlikcore.netlify.com

File: theme/partials/social.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{% if config.extra.social %}
+  <div class="md-footer-social">
+    <link rel="stylesheet" href="{{ base_url }}/assets/fonts/font-awesome.css">
```
Comment (peol) [inline theme/partials/social.html]: This is a pretty heavy dependency for only a couple if small icons IMO

File: theme/partials/social.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    {% for social in config.extra.social %}
+      <a href="{{ social.link }}" class="md-footer-social__link fa fa-{{ social.type }}" target="_blank"></a>
+    {% endfor %}
+        <a href="https://branch.qlik.com" class="md-footer-social__link fa" target="_blank"><img src="{{ base_url }}/images/Icon_Qlik_Branch.png" style="margin-bottom: -5px" height="55px" /></a>
```
Comment (peol) [inline theme/partials/social.html]: Move CSS to CSS file

File: theme/partials/social.html (hunk 1 above)
Comment (hrigner) [reply theme/partials/social.html]: Agree. That's however from mkdocs-material, but we can ofc change it

**Pull request merged**